tgui/packages/tgui/interfaces/CrewConsole.tsx

```diff
@@ -87,9 +98,29 @@ export const CrewConsole = () => {
   );
 };
 
-const CrewTable = (props) => {
-  const { act, data } = useBackend();
-  const { sensors = [] } = data;
+type CrewSensor = {
+  name: string;
+  assignment: string | undefined;
+  ijob: number;
+  life_status: number;
+  oxydam: number;
+  toxdam: number;
+  burndam: number;
+  brutedam: number;
+  area: string | undefined;
+  health: number;
+  can_track: boolean;
```

Comment:
we don't have these, if it's 0 or 1 it's a BooleanLike

Reply:
is there a dedicated type for that, or should i just make it a number?